src/Rector/AbstractPHPUnitRector.php

```diff
@@ -2,21 +2,16 @@
 
 namespace Rector\Rector;
 
-use Nette\Utils\Strings;
 use PhpParser\Node;
+use PHPUnit\Framework\TestCase;
 use Rector\Node\Attribute;
 
 abstract class AbstractPHPUnitRector extends AbstractRector
 {
-    /**
-     * Check if the file is a PHPUnit TestCase. By default, it should end with "Test", as it is the standard.
-     *
-     * @see https://phpunit.de/getting-started-with-phpunit.html
-     */
```

Comment:
Why removing this?

What if sb will come and try to analyze nodes (much worse performance) instead of simple and fast PHPUnit approach.

Reply:
Because I test Rector against an application that didn't follow this rule, plus, all its test classes extends a based one :(

Reply:
I see